projects/1.16/assets/odd-water-mobs/oddwatermobs/lang/zh_cn.json

```
@@ -0,0 +1,212 @@
+  "item.oddwatermobs.raw_ghost_shark": "魔鬼鲨",
+  "item.oddwatermobs.chimaera_sting": "魔鬼鲨刺",
+  "item.oddwatermobs.raw_mudskipper": "生弹涂鱼",
+  "item.oddwatermobs.cooked_mudskipper": "熟弹涂鱼",
+  "item.oddwatermobs.mudskipper_fin": "阿尔法弹涂鱼鳍",
+  "item.oddwatermobs.deep_sea_fish": "深海鱼",
+  "item.oddwatermobs.squat_lobster_tail": "东方扁虾尾",
+  "item.oddwatermobs.cooked_squat_lobster_tail": "熟东方扁虾尾",
+  "item.oddwatermobs.raw_seapig": "海猪",
+  
+  
+  "item.oddwatermobs.bathosphere": "音乐唱片",
+  "item.oddwatermobs.bathosphere.desc": "elgavira - 探海球",
+  "item.oddwatermobs.coelacanth_record": "音乐唱片",
+  "item.oddwatermobs.coelacanth_record.desc": "baggu - 腔棘鱼",
+  "item.oddwatermobs.seapig_record": "音乐唱片",
+  "item.oddwatermobs.seapig_record.desc": "baggu - 海猪",
+  
+  
+  "item.minecraft.potion.effect.sinking": "下沉药水",
+  "item.minecraft.splash_potion.effect.sinking": "喷溅型下沉药水",
+  "item.minecraft.lingering_potion.effect.sinking": "滞留型下沉药水",
+  "item.minecraft.tipped_arrow.effect.sinking": "下沉之箭",
+  "item.minecraft.potion.effect.healing_vigor": "活力药水",
+  "item.minecraft.splash_potion.effect.healing_vigor": "喷溅型活力药水",
+  "item.minecraft.lingering_potion.effect.healing_vigor": "滞留型活力药水",
+  "item.minecraft.tipped_arrow.effect.healing_vigor": "活力之箭",
+  "item.minecraft.potion.effect.potion_of_mudskipping": "弹涂鱼药水",
+  "item.minecraft.potion.effect.potion_of_mudskipping_plus": "弹涂鱼药水",
+  "item.minecraft.splash_potion.effect.potion_of_mudskipping": "喷溅型弹涂鱼药水",
+  "item.minecraft.splash_potion.effect.potion_of_mudskipping_plus": "喷溅型弹涂鱼药水",
+  "item.minecraft.lingering_potion.effect.potion_of_mudskipping": "滞留型弹涂鱼药水",
+  "item.minecraft.lingering_potion.effect.potion_of_mudskipping_plus": "滞留型弹涂鱼药水",
+  "item.minecraft.tipped_arrow.effect.potion_of_mudskipping": "弹涂鱼之箭",
+  "item.minecraft.tipped_arrow.effect.potion_of_mudskipping_plus": "弹涂鱼之箭",
+  "item.minecraft.potion.effect.poison_protection_potion": "抗毒药水",
+  "item.minecraft.splash_potion.effect.poison_protection_potion": "喷溅型抗毒药水",
+  "item.minecraft.lingering_potion.effect.poison_protection_potion": "滞留型抗毒药水",
+  "item.minecraft.tipped_arrow.effect.poison_protection_potion": "抗毒之箭",
+  "item.minecraft.potion.effect.poison_protection_potion_plus": "抗毒药水",
+  "item.minecraft.splash_potion.effect.poison_protection_potion_plus": "喷溅型抗毒药水",
+  "item.minecraft.lingering_potion.effect.poison_protection_potion_plus": "滞留型抗毒药水",
+  "item.minecraft.tipped_arrow.effect.poison_protection_potion_plus": "抗毒之箭",
+  "effect.sinking": "下沉",
+  "effect.healing_vigor": "治愈活力",
+  "effect.poison_protection": "抗毒",
+  
+  
+  
+  "advancements.unknown_seas.title": "未知海域",
+  "advancements.unknown_seas.descr": "从深海中找寻某种东西",
+  "advancements.all_the_way_down.title": "一路到底",
+  "advancements.all_the_way_down.descr": "抵达深海沙丘底部",
+  "advancements.not_in_my_watch.title": "我绝不允许",
+  "advancements.not_in_my_watch.descr": "拯救一条处于窒息中的琵琶鱼",
+  "advancements.overkill.title": "赶尽杀绝",
```

```suggestion
  "advancements.overkill.title": "适得其反",
```
原版曾有有一成就（不是进度）叫赶尽杀绝（Overkill），目标为一次攻击伤害达到9点。但这里不应该是。

---

那是112的，这是116的